dd-sdk-android-ndk/src/main/cpp/utils/stringutils.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+#include "stringutils.h"
+
+#include <jni.h>
+#include <string>
+
+namespace stringutils {
+
+    void copy(JNIEnv *env, jstring from, char *to, size_t max_size) {
```

Comment:
But if you don't expose it anymore, you could maybe just delete it?

Reply:
That was my intention, just forgot to remove it from there too :)